install_config/router/default_haproxy_router.adoc

```diff
@@ -336,6 +336,81 @@ $ openssl ciphers
 Alternatively, use the `ROUTER_CIPHERS` environment variable for an existing
 router.
 
+[[mutual-tls-auth]]
+== Mutual TLS Authentication
+
+Client access to the router and the backend services can be restricted using
+mutual TLS authentication. The router will reject requests from clients not in
+its `authenticated` set.
+Mutual TLS authentication is implemented on client certificates and can be
+controlled based on the certifying authorities (CAs) issuing the certificates,
+the certificate revocation list and/or any certificate subject filters.
+Use the mutual tls config options `--mutual-tls-auth`, `--mutual-tls-auth-ca`,
+`--mutual-tls-auth-crl` and `--mutual-tls-auth-filter` when creating a router:
+
+----
+$ oc adm router --mutual-tls-auth=required  \
+                --mutual-tls-auth-ca=/local/path/to/cacerts.pem   ....
+----
+
+The `--mutual-tls-auth` values are: `required`, `optional`, or `none`, with
+`none` as the default.
+The `--mutual-tls-auth-ca` value optionally specifies a file containing one or
+more CA certificates. These CA certificate[s] are used by the router to verify
+a client's certificate.
+
+The `--mutual-tls-auth-crl` can be optionally used specify the certificate
+revocation list to handle cases where certificates (issued by valid certifying
+authorities) have been revoked.
+
+----
+$ oc adm router --mutual-tls-auth=required  \
+     --mutual-tls-auth-ca=/local/path/to/cacerts.pem  \
+     --mutual-tls-auth-filter='^/CN=my.org/ST=CA/C=US/O=Security/OU=OSE$'  \
+     ....
+----
+
+The optional `--mutual-tls-auth-filter` value can be used for fine grain
+access control based on the certificate subject. The value is a regular
+expression which is to used to match up the certificate's subject.
+
+[NOTE]
+====
+The mutual TLS authentication filter example above shows you a restrictive
+regular expression (regex) - anchored with `^` and `$` - that `exactly` matches
+a certificate subject.  If you decide to use a less restrictive regular
+expression, please be aware that this can potentially match certificates
+issued by any certifying authorities (CAs) you have deemed to be valid.
```

Comment:
s/certifying authorities (CAs)/CAs

Since the acronym was previously defined in the this topic.